src/components/ExampleComponents.tsx

```diff
@@ -33,3 +34,6 @@ addDisplayNames(Heading, OrigHeading, 'Heading');
 
 export class Text extends OrigText {}
 addDisplayNames(Text, OrigText, 'Text');
+
+export class TextInput extends OrigTextInput {}
+addDisplayNames(Text, OrigText, 'TextInput');
```

Comment:
addDisplayNames(**Text**, OrigText, 'TextInput');
--> it should be **TextInput** instead?

Reply:
It should. Thanks. Fixed it.